versioned_docs/version-v6.0.0/framework/pubsubV2.md

```diff
@@ -4,28 +4,41 @@ sidebar_label: Publish and subscribe
 
 # PubSub V2
 
-## Motivation
-Asynchronous messaging helps with decoupling publishers from consumers, as it avoids the blocking during publishing. Especially in LogicApps workflows, where the publisher and consumer is not always synchronous.
+:::note[motivation]
+Asynchronous messaging helps with decoupling publishers from consumers, as it avoids the blocking during publishing. Especially in Logic Apps workflows, where the publisher and consumer is not always synchronous.
 
-Invictus provides a solution called the PubSub, that allows Azure Service Bus to act as a message broker and interact in a publish/subscribe-approach via HTTP endpoints.
+Invictus provides a solution called the PubSub, that allows Azure Service Bus to act as a message broker and interact in a publish/subscribe-approach via HTTP endpoints; plus having Azure Blob Storage act as a claim-check provider for when messages are to big to be published directly.
 
-## Usage
-The PubSub is available as a HTTP endpoint in your LogicApp workflow.
+*🔗 See also the [Publisher-Subscriber integration pattern](https://learn.microsoft.com/en-us/azure/architecture/patterns/publisher-subscriber).*
+:::
 
-- Default topic name: `pubsubv2router`.
-- Default claim-checked messages blob container name: `invictuspubsubv2router`.
-- Subscription filter rules are assigned the same name as the subscription.
-- Default message time to live: `30 days`
-- Default subscription lock timeout: `1 minute`
+## Available endpoints
+* [`/api/Publish`](#️-publish-single-message): by sending a HTTP request with a custom content, it places a message on an Azure Service Bus topic.
+* [`/api/Subscribe`](#️-subscribe-for-messages): by sending a HTTP request with a specified Azure Service Bus topic subscription name, it response available messages.
+* [`/api/Acknowledge`](#-acknowledge-message): by sending a HTTP request with a specified message sequence number, it settles the Azure Service Bus message.
```

Comment:
this link doesnt seem to work

Reply:
Aha, thx for noticing, fixed it here: https://github.com/invictus-integration/docs-ifa/pull/378/commits/62dd14cab594e065e3c7ed4d1045faab82c7a611